docs/github-labels.md

```diff
@@ -0,0 +1,108 @@
+| kind/failing-test                  | Categorizes issue or PR as related to a consistently or frequently failing test | Any |
+| kind/feature                       | Categorizes issue or PR as related to a new feature.      | Any                |
+| kind/support                       | Categorizes issue or PR as related to a support question. | Any |
+| lifecycle/active                   | Indicates that an issue or PR is actively being worked on by a contributor. | Any |
+| lifecycle/frozen                   | Indicates that an issue or PR should not be auto-closed due to staleness. | Any |
+| lifecycle/rotten                   | Denotes an issue or PR that has aged beyond stale and will be auto-closed. | Any |
+| lifecycle/stale                    | Denotes an issue or PR has remained open with no activity and has become stale. | Any |
+| priority/awaiting-more-evidence    | Lowest priority. Possibly useful, but not yet enough support to actually get it done. | Any |
+| priority/backlog                   | Higher priority than priority/awaiting-more-evidence. | Any |
+| priority/critical-urgent           | Highest priority. Must be actively worked on as someone's top priority right now. | Any |
+| priority/important-longterm        | Important over the long term, but may not be staffed and/or may need multiple releases to complete. | Any |
+| priority/import-soon               | Must be staffed and worked on either currently, or very soon, ideally in time for the next release. | Any |
+| ready-to-work                      | Indicates that an issue or PR has been sufficiently triaged and prioritized and is now ready to work. | Any |
+| size/L                             | Denotes a PR that changes 100-499 lines, ignoring generated files. | Any |
+| size/M                             | Denotes a PR that changes 30-99 lines, ignoring generated files.| Any |
+| size/S                             | Denotes a PR that changes 10-29 lines, ignoring generated files.| Any |
+| size/XL                            | Denotes a PR that changes 500+ lines, ignoring generated files.| Any |
+| size/XS                            | Denotes a PR that changes 0-9 lines, ignoring generated files.| Any |
+| triage/duplicate                   | Indicates an issue is a duplicate of other open issue. | Humans |
+| triage/needs-information           | Indicates an issue needs more information in order to work on it. | Humans |
+| triage/not-reproducible            | Indicates an issue can not be reproduced as described. | Humans |
+| triage/unresolved                  | Indicates an issue that can not or will not be resolved. | Humans |
+
+## Labels that apply only to issues
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| good first issue                   | Denotes an issue ready for a new contributor, according to the "help wanted" guidelines. | Anyone |
```

Comment:
I don't think you meant "help wanted" here - but maybe "good first issue"?

Reply:
That was a copy from k/k. These labels represent two separate intentions. See https://kubernetes-contributor.netlify.com/guide/help-wanted/ and let me know if you think this is useful.